sig-security/security-audit-2021/RFP.md

```diff
@@ -0,0 +1,121 @@
+
+### Eligible Vendors
+
+This RFP is open to proposals from all vendors.
+
+#### Constraints
+
+If your proposal includes subcontractors, please include relevant details from their firms such as CVs, past works, etc. The selected vendor will be wholly responsible for fulfillment of the audit and subcontractors must be wholly managed by the selected vendor.
+
+### Anticipated Selection Schedule
+
+This RFP will be open between 2021/01/25 and 2021/02/26.
+
+The working group will answer questions for the first two weeks of this period.
+
+Questions can be submitted [here](https://docs.google.com/forms/d/e/1FAIpQLScjApMDAJ5o5pIBFKpJ3mUhdY9w5s9VYd_TffcMSvYH_O7-og/viewform). All questions will be answered publicly in this document.
+
+Proposals must include CVs, resumes, and/or example reports from staff that will be working on the project.
+
+Proposals should be submitted to kubernetes-security-audit-2021@googlegroups.com
+
+* 2021/01/25: RFP Open, Question period open
+* 2021/02/08: Question period closes
+* 2021/02/26: RFP Closes
+* 2021/03/09: The working group will announce vendor selection
+
+## Methodology
+
+We are allowing roughly 12 calendar weeks for this audit, start date can be negotiated after vendor selection. 
+
+The working group will establish a 60 minute kick-off meeting to answer any initial questions and discuss the Kubernetes architecture.
+
+This is a comprehensive audit, not a penetration test or red team exercise. The audit does not end with the first successful exploit or critical vulnerability.
+
+The vendor will document the Kubernetes configuration and architecture that they will audit and provide this to the working group. The cluster deployment assessed must not be specific to any public cloud. The working group must approve this configuration before the audit continues. This documented configuration will result in the "audited reference architecture specification" deliverable.
+
+The vendor will perform source code analysis on the Kubernetes code base, finding vulnerabilities and, where possible and making the most judicious use of time, providing proof of concept exploits that the Kubernetes project can use to investigate and fix defects. The vendor will discuss findings on a weekly basis and, at the vendor’s discretion, bring draft write-ups to status meetings.
+
+The working group will be available weekly to meet with the selected vendor and will provide subject matter experts as requested.
+
+The vendor will develop and deliver a draft report, describing their methodology, how much attention the various components received (to inform future work), and the work’s findings. The working group will review and comment on the draft report, either requesting updates or declaring the draft final. This draft-review-comment-draft cycle may repeat several times.
+
+## Expectations
+
+The vendor must report urgent security issues immediately to both the working group and security@kubernetes.io.
```

Comment:
@kubernetes/product-security-committee does this look fine to y'all with respect to our embargo policy?

Reply:
I would love to hear from the @kubernetes/product-security-committee. FWIW, this is the process we used last time.